homeassistant/components/template/light.py

```diff
@@ -47,7 +48,7 @@
 )
 
 _LOGGER = logging.getLogger(__name__)
-_VALID_STATES = [STATE_ON, STATE_OFF, "true", "false"]
+_VALID_STATES = [STATE_ON, STATE_OFF, STATE_UNKNOWN, "true", "false", "none"]
```

Comment:
Jinja2's `{{ None }}` will render to a `"none"` value.

Reply:
I think this is because of the string casting at 568. The `None` value should be checked before that (just like the boolean value is checked sooner).

Reply:
That was it. Thank you.